netebpfext/net_ebpf_ext.h

```diff
@@ -223,7 +232,9 @@ net_ebpf_extension_add_wfp_filters(
  */
 void
 net_ebpf_extension_delete_wfp_filters(
-    uint32_t filter_count, _Frees_ptr_ _In_count_(filter_count) net_ebpf_ext_wfp_filter_id_t* filter_ids);
+    _In_ net_ebpf_extension_wfp_filter_context_t* filter_context,
+    uint32_t filter_count,
```

Comment:
The last 2 parameters are redundant if you passed filter_context as the first param. If you want to keep the parameters, then pass in the engine handle instead of the filter context.

Reply:
I prefer 'need-to-know' :-) Fixed to pass in the engine handle instead.